lib/src/component_declaration/annotations.dart

```diff
@@ -243,12 +243,16 @@ class Accessor {
   /// The error message displayed when the accessor is not set.
   final String requiredErrorMessage;
 
+  /// Whether to skip generating an accessor for this field.
+  final bool doNotGenerate;
```

Comment:
Should we add a constant annotation for this? `@doNotGenerate` obviously not needed for this PR but might be nice for later

Reply:
Yeah, I thought about adding one but decided against it since this won't be used very often (we haven't had a need for it until now).